test/Microsoft.AspNetCore.SignalR.Client.Tests/TestConnection.cs

```diff
@@ -116,9 +116,23 @@ public Task ReceiveBytesAsync(byte[] bytes)
         {
             return Application.Output.WriteAsync(bytes).AsTask();
         }
-        public async Task<string> ReadSentTextMessageAsync()
+        public async Task<string> ReadSentTextMessageAsync(bool ignorePings=true)
         {
             // Read a single text message from the Application Input pipe
+
+            while (true)
+            {
+                var result = await ReadSentTextMessageAsyncInner();
+                if (ignorePings && result == "{\"type\":6}")
```

Comment:
Add a `IsPingMessage` method for testing whether content is a ping or not.

Load string into a `JObject`, then test if the "type" property is 6. That will be more resilient to any unit tests that change the formatting to include spaces or line breaks in JSON messages.